python/ray/data/_internal/arrow_ops/transform_pyarrow.py

```diff
@@ -1,5 +1,10 @@
 from typing import TYPE_CHECKING, List, Union
 
+from ray.air.util.tensor_extensions.arrow import (
```

Comment:
looks like this import causing CI failure of 

```
File "/ray/python/ray/data/_internal/arrow_ops/transform_pyarrow.py", line 3, in <module>
--
  | from ray.air.util.tensor_extensions.arrow import (
  | File "/ray/python/ray/air/util/tensor_extensions/arrow.py", line 6, in <module>
  | import pyarrow as pa
  | ModuleNotFoundError: No module named 'pyarrow'
```

Can this import be put into `unify_schemas()` method instead?

Reply:
Yep this import needs to be deferred until runtime rather than import time!